.circleci/config.yml

```diff
@@ -38,9 +38,10 @@ jobs:
             - ~/.cache/yarn
       - run: mkdir -p ./test-results
       - run:
-          command: ./script/run-isolated-tests.sh
```

Comment:
This was a bit fiddly to get right and didn't do what we wanted on master (duh, master changed nothing compared to master). And reducing unit test time in Circle doesn't matter much since the Jenkins pipeline is the critical path anyway. We'll properly isolate when we get to app splitting.